iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -767,12 +756,6 @@ public Identifier IDENTIFIER() {
         b.token(BicepLexicalGrammar.IDENTIFIER_LITERAL)));
   }
 
-  public Identifier IDENTIFIER_NOT_KEYWORD() {
-    return b.<Identifier>nonterminal(BicepLexicalGrammar.IDENTIFIER_NOT_KEYWORD).is(
```

Comment:
This `BicepLexicalGrammar.IDENTIFIER_NOT_KEYWORD` should also be removed from `BicepLexicalGrammar`